src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -379,6 +378,10 @@ private static void makePublic(final CtClass c) {
    * @throws InvocationTargetException if the constructor of a class throws an
    *   exception
    * @throws BadBytecode if the on-the-fly compilation uses an invalid bytecode
+   *
+   * In the case of an `ExceptionInInitializerError` from `newInstance`, which
```

Comment:
Again, a bit confusing that this didn't happen in the previous commit.

Reply:
I can squash commits later if you want.